src/components/dropdown/README.md

```diff
@@ -66,6 +66,7 @@ const handleChange = (data) => console.log(data);
 | ---- | ---- | ------- | ----------- |
 | options | array | *required* | Array of objects. Check [options notes](#options) to further details on this prop |
 | renderTrigger | function || A function to render a custom dropdown trigger. Check [renderTrigger notes](#renderTrigger) to further details |
+| renderOption | function || A function to render a custom option |
```

Comment:
Nifty pick: can you add a space in the empty column? I think it’s like that in other components. Please change everywhere in this PR.